playwright_tests/Toaster.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
```

Comment:
this should import from `playwright-test-coverage` like the other tests so that the test coverage will work